src/test/lua/exasolrls/test_rls_adapter.lua

```diff
@@ -43,11 +48,27 @@ function test_rls_adapter.test_create_virtual_schema()
 end
 
 function test_rls_adapter.test_get_capabilites()
+    local request = {}
     local expected = {type = "getCapabilities",
         capabilities = {"SELECTLIST_PROJECTION", "AGGREGATE_SINGLE_GROUP", "AGGREGATE_GROUP_BY_COLUMN",
             "AGGREGATE_GROUP_BY_TUPLE", "AGGREGATE_HAVING", "ORDER_BY_COLUMN", "LIMIT",
             "LIMIT_WITH_OFFSET"}}
-    local actual = adapter.get_capabilities()
+    local actual = adapter.get_capabilities(nil, request)
+    luaunit.assertEquals(actual, expected)
+end
+
+function test_rls_adapter.test_get_capabilites_while_excluding_capabilities()
+    local request = {
+        schemaMetadataInfo = {
+            properties = {
+                EXCLUDED_CAPABILITIES = "AGGREGATE_GROUP_BY_COLUMN, AGGREGATE_GROUP_BY_TUPLE, AGGREGATE_HAVING"
```

Comment:
Is there a space between the last two or not? It's just not clear with this preview.
I just want to make sure we also test `a,b` case, without spaces

Reply:
Added variants to check robustness.